src/main/java/EchoServer.java

```diff
@@ -0,0 +1,29 @@
+import echoserver.ClientConnection;
+import echoserver.Console;
+import echoserver.HostServer;
+import echoserver.Presenter;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+
+public class EchoServer {
+    public static void main(String[] args) throws IOException {
+        Presenter hostPresenter = new Presenter(new Console(System.out));
+        ServerSocket ss = new ServerSocket(100);
+        HostServer hs = new HostServer(ss);
```

Comment:
You might look into try with resources.

https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html